components/dropdown/README.md

```diff
@@ -209,6 +209,11 @@ To make your usage of `d2l-dropdown-more` accessible, use the following property
 * `footer`: Sticky container at the bottom of the dropdown
 * `default`: Anything inside of `d2l-dropdown-content` that isn't in the `header` or `footer` slots appears as regular content
 
+**Methods:**
+* `async resize()`: Call if the size of the content changes due to a change in
+a nested component. The nested component may choose to fire a custom event, which
+the component containing the `d2l-dropdown-content` can catch and call this method.
```

Comment:
Nit: I'm not sure if mentioning custom events brings any value since calling `resize` from an event listener is more about one specific use case than how it could be used in general. Its really just a public method that triggers the `dropdown-content` to resize to fit its content. It could also get called due to things like a `MutationObserver`, `ResizeObserver` or even something else altogether.